specification/authorization/resource-manager/Microsoft.Authorization/preview/2018-05-01-preview/authorization-AccessReviewCalls.json

```diff
@@ -1026,6 +1026,474 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Authorization/roleAssignmentApprovals": {
+      "get": {
+        "tags": [
+          "ListRoleAssignmentApprovals"
+        ],
+        "operationId": "RoleAssignmentApproval_List",
+        "x-ms-examples": {
+          "GetAccessReviews": {
+            "$ref": "./examples/GetRoleAssignmentApprovals.json"
+          }
+        },
+        "description": "Get role assignment approvals.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Describe the result of a successful operation.",
+            "schema": {
+              "$ref": "#/definitions/RoleAssignmentApprovalListResult"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/ErrorDefinition"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Authorization/roleAssignmentApprovals/{approvalId}": {
+      "get": {
+        "tags": [
+          "GetRoleAssignmentApprovalById"
```

Comment:
I don't see any APIs to create approvals. I assume they created behind the scenes, but what actually triggers their creation? And how is it configured?

Reply:
And how does the chain of approvers get configured?

---
In reply to: [551591962](https://github.com/Azure/azure-rest-api-specs/pull/12108#discussion_r551591962) [](ancestors = 551591962)

Reply:
Yes, approvals are created behind the scenes when a roleAssignmentRequest is created. I've added a PDF to the PR description which documents the flow and how approvals are created. I'll clarify more details in the meeting.